src/views/Home/Home.jsx

```diff
@@ -1,14 +1,28 @@
 import React, { useState, useEffect } from 'react';
 import { getToken } from '@the-collab-lab/shopping-list-utils';
+import { useNavigate } from 'react-router-dom';
+import { db } from '../../lib/firebase';
+import { collection, getDocs, query } from 'firebase/firestore';
 import CreateList from '../../components/Home/CreateList';
 import JoinList from '../../components/Home/JoinList';
 
-const Home = ({ token, setToken }) => {
-  const [hasToken, setHasToken] = useState(token !== null);
+const Home = ({ token, setToken, hasToken, setHasToken }) => {
+  const [joinList, setJoinList] = useState();
+  const [formError, setFormError] = useState();
+
+  const navigate = useNavigate();
+
+  const handleChange = (e) => {
+    setJoinList(e.target.value);
+  };
 
   useEffect(() => {
     const token = localStorage.getItem('token');
     setHasToken(token !== null);
+    const navigateToList = () => navigate(`/list`);
+    if (hasToken) {
+      navigateToList();
+    }
   }, []);
```

Comment:
🧹 Just a note for some code-cleanliness: 
- This hook should probably be made aware of its dependencies (right now, I think that's `token` and `navigate`)
- Do we need `navigateToList` or can we just call `navigate` in our condition here?

Reply:
Made suggested changes (thank you!).